### PR TITLE
HYDRA-889 - Create a unit test for the shaded display mode with a texture image file

### PR DESCRIPTION
**Pull request by vlasovi**

### CODE REVIEW

**Comment by vlasovi** [comment]
FYI : @lilike-adsk @ppt-adsk @roopavr-adsk @debloip-adsk

File: test/lib/mayaUsd/render/mayaToHydra/testUVandUDIM.py [hunk 1]
```diff
@@ -39,5 +39,11 @@ def test_UDIMs(self):
         self.setHdStormRenderer()
         self.verifySnapshot("quads_with_UDIMs.png")
 
+    def test_HDR(self):
+        mayaUtils.openTestScene("testUVandUDIM", "testHDR.ma")
+        self.setHdStormRenderer()
+        cmds.refresh()
+        self.assertSnapshotClose("quads_with_HDR.png", 0.04, 0.2)
```
Comment (debloip-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/testUVandUDIM.py]: For consistency, I would suggest we either use and adjust the existing threshold constants, or if we use bespoke thresholds per-test as here, then we should have the other test follow the same structure as well and adjust its code to mirror this one

File: test/testSamples/testUVandUDIM/testHDR.ma [hunk 2]
```diff
@@ -0,0 +1,552 @@
+//Maya ASCII 2026ff01 scene
```
Comment (debloip-adsk) [inline test/testSamples/testUVandUDIM/testHDR.ma]: Year needs to be changed, Maya 2026 is not a released product

File: test/testSamples/testUVandUDIM/testHDR.ma [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+//Name: testHDR.ma
+//Last modified: Thu, Jul 18, 2024 02:05:49 PM
+//Codeset: 1252
+requires maya "2026ff01";
```
Comment (debloip-adsk) [inline test/testSamples/testUVandUDIM/testHDR.ma]: Year needs to be changed, Maya 2026 is not a released product

**Review by debloip-adsk — CHANGES_REQUESTED** [review]

**Review by debloip-adsk — APPROVED** [review]

**Pull request merged**